src/components/Navbar/Navbar.js

```diff
@@ -360,6 +362,15 @@ const MobileNav = props => (
             <FormattedMessage {...messages.metrics} />
           </NavLink>
         </li>
+        {
+          props.isSuperUser
+            ? <li>
+                <NavLink to="/superadmin" onClick={props.closeMobileMenu}>
+                  <FormattedMessage {...messages.superAdmin} />
+                </NavLink>
+              </li>
+            : null
+        }
```

Comment:
didn't we add this in v1?  why is v1 included in this PR?

Reply:
I don't think we have a navbar specifically for mobile in v1.